cmake/onnxruntime-win-x64-directml.cmake

```diff
@@ -0,0 +1,72 @@
+# Copyright (c)  2022-2023  Xiaomi Corporation
+message(STATUS "CMAKE_SYSTEM_NAME: ${CMAKE_SYSTEM_NAME}")
+message(STATUS "CMAKE_SYSTEM_PROCESSOR: ${CMAKE_SYSTEM_PROCESSOR}")
+message(STATUS "CMAKE_VS_PLATFORM_NAME: ${CMAKE_VS_PLATFORM_NAME}")
+
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Windows)
+  message(FATAL_ERROR "This file is for Windows only. Given: ${CMAKE_SYSTEM_NAME}")
+endif()
+
+if(NOT (CMAKE_VS_PLATFORM_NAME STREQUAL X64 OR CMAKE_VS_PLATFORM_NAME STREQUAL x64))
+  message(FATAL_ERROR "This file is for Windows x64 only. Given: ${CMAKE_VS_PLATFORM_NAME}")
+endif()
+
+if(BUILD_SHARED_LIBS)
```

Comment:
Is it only for share libraries?

Reply:
Not sure. How can I find out? Should I omit that error statement?